### PR TITLE
Generating new Jest Mocks

### PR DESCRIPTION
**Pull request by Szymon20000**
# Why

https://github.com/expo/expo/issues/5564

# How

https://github.com/expo/expo/blob/master/guides/Generating%20Jest%20Mocks.md

# Test Plan

?

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12370,30 +12370,6 @@ react-native-tab-view@^1.0.0, react-native-tab-view@^1.2.0, react-native-tab-vie
   dependencies:
     prop-types "^15.6.1"
 
-react-native-unimodules@^0.5.3, react-native-unimodules@~0.5.0, react-native-unimodules@~0.5.2:
```
Comment (bbarthec) [inline yarn.lock]: This one is interesting 🤔 I would rather not remove this package

**Review by bbarthec — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (sjchmiela) [reply yarn.lock]: I think it is rightfully removed after initializing `react-native-unimodules` submodule, isn't it?

Comment (sjchmiela) [reply yarn.lock]: But I agree fixing `master` is a bit out of the scope of this PR, so it can stay like it is now

**Review by sjchmiela — APPROVED** [review]
Thank you, great job!

**Review by tsapeta — APPROVED** [review]
👍

File: yarn.lock (hunk 1 above)
Comment (ide) [reply yarn.lock]: we've removed it in another PR. the issue is what @sjchmiela identified. perhaps in direnv we could run `git submodule init` to make this less of an issue. not sure we want to run `git submodule update` but at least `init` makes `update` do the right thing

**Pull request merged**